elisp/intero.el

```diff
@@ -2660,6 +2666,14 @@ For debugging purposes, try running the following in your terminal:
         (setq intero-ghc-version
               (intero-ghc-version-raw)))))
 
+(defun intero-ghci-output-flags ()
+  "Get the appropriate ghci output flags for the current GHC version"
+  (with-current-buffer (intero-buffer 'backend)
+    (let ((current-version (mapcar #'string-to-number (split-string intero-ghc-version "\\."))))
```

Comment:
Do I get away with the direct reference to the variable here? I could call out to the function instead

Reply:
That looks fine.